nav_msgs/Path waypoints_path
---
nav_msgs/Path spline_path
float64[] sd_steps 
float64[] cumulative_sd_steps